external/common/include/fast_cpp_csv_parser/LICENSE

```diff
@@ -0,0 +1,28 @@
+Copyright (c) 2015, ben-strasser
```

Comment:
Oops we were talking about two different things I think. If you want to keep the README from the csv project then put it in the same directory as the `csv.h` file. I was thinking of just having `external/common/includes/README.md` be a file that we write that says where `csv.h` came from (with a URL). But I think you are right that the csv.h README.md would be good to keep since it has usage info and such so `external/common/includes/fast_cpp_csv_parser/README.md` would be a good spot for that.